includes/class-wc-order-item-product.php

```diff
@@ -167,7 +167,7 @@ public function set_taxes( $raw_tax_data ) {
 	 */
 	public function set_variation( $data ) {
 		foreach ( $data as $key => $value ) {
-			$this->add_meta_data( str_replace( 'attribute_', '', $key ), $value, true );
```

Comment:
That prefix was getting removed, and I modified it to keep the prefix, as that was the best way to reliably determine which meta values were attributes. For my clarity, does having the prefix or not having the prefix break backwards compatibility?